src/ansys/geometry/core/designer/designpoint.py

```diff
@@ -86,3 +88,9 @@ def __repr__(self) -> str:
         lines.append(f"  Name                 : {self.name}")
         lines.append(f"  Design Point         : {self.value}")
         return "\n".join(lines)
+
+    def _to_polydata(self) -> pv.PolyData:
+        """Get polydata from DesignPoint object."""
+        # get units to plot proportionally
+        unit = 0.3 * self.value.unit
```

Comment:
0.3 seems kind of a magic number. Can we explain the rationale?

Reply:
Honestly, I had to select a size for the sphere representation of the points. I think the default value of the sphere was too big, so I reduced its size. 

Anyway, we should be representing the points with a decimal of the original unit, so we don't hide full units under the sphere while keeping the representation of the point visible in the plotter.

I can't think of a better approach to this, if you have an idea of how to approach this better please let me know 🙂

Reply:
Adding a comment saying precisely this, is more than enough =)

> 0.3 = size for the sphere representation of the points - determined empirically for proper representation